JS.md

```diff
@@ -377,13 +377,21 @@ Return a new `WebAssembly.Memory` instance with [[Memory]] set to `m` and
 
 ### `WebAssembly.Memory.prototype.grow`
 
+The `grow` method has the signature:
+```
+grow(delta)
+```
+
 Let `M` be the `this` value. If `M` is not a `WebAssembly.Memory`,
 a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror)
 is thrown.
 
-This method performs a [`grow_memory`](AstSemantics.md#resizing) on
-`M.[[Memory]]`, having first performed [`ToInteger`](http://tc39.github.io/ecma262/#sec-tointeger)
-on the first argument. On failure, a `WebAssembly.RuntimeError` is thrown.
+Let `d` be [`ToInteger`](http://tc39.github.io/ecma262/#sec-tointeger)(`delta`).
```

Comment:
So the goal here (and in the 3 other places that use `ToInteger`) is not to forcibly coerce everything via `ToUint32` since that prevents you from ever allowing values >2<sup>32</sup> and we may regret that when we do wasm64.  However, what I'm implicitly leaving out is the range error when the result of `ToInteger` is negative or crazy-big; I was sorta implicitly relying on the dependent operation to reject these values.  I could make another PR to factor out a new `ToUnsignedInteger` that is used by all four places, if that makes sense.

Also probably want a separate PR to switch all the http:// to https:// in this file (there are many).